services/workflows-service/src/providers/file/utils/extract-base64-payload.ts

```diff
@@ -0,0 +1,3 @@
+export const extractBase64Payload = (base64: string): string => {
+  return base64.split(',')[1] as string;
+};
```

Comment:
This function should either handle invalid strings and non-strings or just return the undefined that is expected here.
```suggestion
export const extractBase64Payload = (base64: string) => {
  return base64.split(',')[1];
};
```

Reply:
P1

Reply:
Non relevant anymore.